fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -289,6 +300,30 @@ def sst_dst_to_dense(self, sst: Tensor, dst: Optional[Tensor] = None) -> Tensor:
             dense_rt += dst
         return dense_rt
 
+    def lossy_compress(self, dense: Tensor) -> Tuple[Tensor, Tensor, Tensor]:
+        """From dense tensor to lossy reconstruction of dense tensor with the help of SST and DST
+        tensor calculation. If requested sparsity is zero (or top_100_percent) then simply returns
+        the input dense tensor as the reconstruction.
+
+        Args:
+            dense (Tensor):
+                Input dense tensor (no zeros).
+
+        Returns:
+            (Tuple[Tensor, Tensor, Tensor]):
+                A tuple of the form (lossy_reconstruction, sst, dst) with three tensors of the same
+                shape as the dense tensor.
+        """
+        # when sparsity is 0% for both sst and dst the dense tensor itself is returned
```

Comment:
move this to before line 321 and explain more clearly why sst is none and dst is the original pls.